cli/command/container/opts.go

```diff
@@ -660,7 +660,7 @@ func parse(flags *pflag.FlagSet, copts *containerOptions, serverOS string) (*con
 	hostConfig := &container.HostConfig{
 		Binds:           binds,
 		ContainerIDFile: copts.containerIDFile,
-		OomScoreAdj:     copts.oomScoreAdj,
+		OomScoreAdj:     int(copts.oomScoreAdj),
```

Comment:
Both sides should already be an `int` I think, so this change should probably be removed.